.cloudbuild.yaml

```diff
@@ -211,10 +211,10 @@ steps:
   args: ['pull', 'gcr.io/cloudsql-docker/gce-proxy:1.14']
   id:   'pullCloudsqlProxy'
 - name: 'gcr.io/cloud-builders/docker'
-  args: ['pull', 'gcr.io/ml-pipeline/argoexec:v2.3.0-license-compliance']
+  args: ['pull', 'gcr.io/ml-pipeline/argoexec:v2.7.5-license-compliance']
```

Comment:
FYI, the suffix `-license-compliance` was added to distinguish it from `gcr.io/ml-pipeline/argoexec:v2.3.0`, there's no longer such a need because `gcr.io/ml-pipeline/argoexec:v2.7.5` doesn't exist, so we can just use that name.

Not sure if you intentionally kept the suffix, I'd prefer removing it.

Reply:
The scripts that add licenses add this suffix. I think the suffix has advantage of clearly marking the image as different from the original Argo image.

Reply:
SGTM, if you find the extra information helpful